docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sphinx==6.1.3
+sphinx_rtd_theme==1.2.0
+sphinx-autoapi==2.0.1
```

Comment:
I don't think we need the `sphinx-autoapi`, it was for automatically generating docs from docstrings. And I don't think that will be applicable for this documentation.